EyeDropper/explainer.md

```diff
@@ -40,55 +39,31 @@ Additionally, browsers should provide a clear indication as to when the user has
 The transition into eyedropper mode should require [consumable user activation](https://github.com/mustaqahmed/user-activation-v2), for example, clicking on a button from the web page, to help avoid unintentionally revealing pixel data.
 
 ## Solution
```

Comment:
I think we need to add something about color space.  It should say:

1. the representation of color on the web is in transition
2. this working group is developing an object to represent color
3. to avoid competing with other efforts this API will initially provide a single color string which is gamut mapped to the sRGB color space
4. it is expected that a color object will be the primary mechanism by which authors will access sampled color data in the future and that it will have the facilities to map between color spaces